CompareImage.cpp

```diff
@@ -202,16 +205,18 @@ void hash_function(int start_point)
 					{
 						hasher = hasher + gray_image.getMat(cv::ACCESS_READ).at<uchar>(l, j); //will leave it like this for now, seems to work
 					}
-					hasher = hasher * hasher; //just to make hasher even more unique
+					hasher = hasher * hasher; //just to make hasher even more unique //FIXME does it really?

```

Comment:
Yes, imagine there's 2 hashes that are very close together, if you multiply one for another they are gonna be further away from each other